Dockerfile

```diff
@@ -82,8 +60,8 @@ RUN mkdir -p /data/data /data/log
 
 VOLUME /data
 
-ADD config/crate.yml /crate/config/crate.yml
-ADD config/log4j2.properties /crate/config/log4j2.properties
+COPY --chown=1000:0 config/crate.yml /crate/config/crate.yml
+COPY --chown=1000:0 config/log4j2.properties /crate/config/log4j2.properties
 COPY docker-entrypoint.sh /
```

Comment:
Consider to do this step before the actual installation of crate and crash as this is causing a late layer although it is changing very rarely.